src/Reservoir.jl

```diff
@@ -138,13 +137,13 @@ function grad_reservoir_storage_captures(m::Model)
 end
 
 function constraintoffset_reservoir_storagecapacitymin(m::Model)
-    gen(rr) = m.external_parameters[:storagecapacitymin].values[rr]
-    hallsingle(m, :Reservoir, :storage, gen, [:time])
+    gen(rr, tt) = m.external_parameters[:storagecapacitymin].values[rr]
+    hallsingle(m, :Reservoir, :storage, gen)
```

Comment:
The current code is correct.  It should only have r, because it's duplicated over t.

Reply:
If I don't do this, it indeeds duplicates over t, but in the wrong "order/dimension", resulting in the wrong order in the b vector: [value[rr=1, tt=1] value[rr=1, tt=2] ... value[rr=2, tt=1]] instead of looping for on rr and then on tt ...

Reply:
Ah, but this is a bug in OptiMimi!  I just fixed it.  Can you update (`Pkg.checkout("OptiMimi")`) and test?

Reply:
It works !